src/views/product-tutorials-view/product-tutorials-view.module.css

```diff
@@ -1,3 +1,8 @@
+.heading {
+  composes: g-offset-scroll-margin-top from global;
+  composes: g-screen-reader-only from global;
```

Comment:
The top-level heading on the page isn't shown in mockups, only in the table of contents. With this in mind, we hide the top-level heading, except from screenreaders.

This allows the table of contents to work as expected, while still matching mockups.